extensions/debug-auto-launch/src/extension.ts

```diff
@@ -8,6 +8,12 @@ import { createServer, Server } from 'net';
 import * as vscode from 'vscode';
 import * as nls from 'vscode-nls';
 
+const enum State {
+	Disabled = 'disabled',
+	OnlyWithFlag = 'onlyWithFlag',
+	Smart = 'smart',
+	Always = 'always',
+}
```

Comment:
fyi @weinand @connor4312 - this const enum needed to move above it usage because transpile currently doesn't inline them